SampleCode/Python/sample_identify_cross_page_tables.ipynb

```diff
@@ -128,28 +145,51 @@
+    "    list: A concision list of result.tables. The significance is to store the calculated data to avoid repeated calculations in subsequent reference.\n",
     "    \"\"\"\n",
+    "    table_concision_list = []\n",
     "    merge_tables_candidates = []\n",
     "    pre_table_idx = -1\n",
     "    pre_table_page = -1\n",
     "    pre_max_offset = 0\n",
     "\n",
     "    for table_idx, table in enumerate(tables):\n",
     "        min_offset, max_offset = get_table_span_offsets(table)\n",
-    "        table_page = min(get_table_page_numbers(table))\n",
-    "        \n",
-    "        # If there is a table on the next page, it is a candidate for merging with the previous table.\n",
-    "        if table_page == pre_table_page + 1:\n",
-    "            pre_table = {\"pre_table_idx\": pre_table_idx, \"start\": pre_max_offset, \"end\": min_offset}\n",
-    "            merge_tables_candidates.append(pre_table)\n",
-    "        \n",
-    "        print(f\"Table {table_idx} has offset range: {min_offset} - {max_offset} on page {table_page}\")\n",
+    "        if min_offset > -1 and max_offset > -1:\n",
+    "            table_page = min(get_table_page_numbers(table))\n",
+    "            print(f\"Table {table_idx} has offset range: {min_offset} - {max_offset} on page {table_page}\")\n",
+    "\n",
+    "            # If there is a table on the next page, it is a candidate for merging with the previous table.\n",
+    "            if table_page == pre_table_page + 1:\n",
+    "                pre_table = {\n",
+    "                    \"pre_table_idx\": pre_table_idx,\n",
+    "                    \"start\": pre_max_offset,\n",
+    "                    \"end\": min_offset,\n",
+    "                    \"min_offset\": min_offset,\n",
+    "                    \"max_offset\": max_offset,\n",
+    "                }\n",
+    "                merge_tables_candidates.append(pre_table)\n",
+    "                \n",
+    "            table_concision_list.append(\n",
+    "                {\n",
+    "                    \"idx\": table_idx,\n",
+    "                    \"min_offset\": min_offset,\n",
+    "                    \"max_offset\": max_offset,\n",
+    "                    \"page\": table_page,\n",
+    "                }\n",
+    "            )\n",
     "\n",
-    "        pre_table_idx = table_idx\n",
-    "        pre_table_page = table_page\n",
-    "        pre_max_offset = max_offset\n",
+    "            pre_table_idx = table_idx\n",
+    "            pre_table_page = table_page\n",
+    "            pre_max_offset = max_offset\n",
+    "        else:\n",
+    "            print(f\"Table {table_idx} is empty\")\n",
+    "            table_concision_list.append(\n",
+    "                {\"idx\": {table_idx}, \"min_offset\": -1, \"max_offset\": -1, \"page\": -1}\n",
+    "            )\n",
     "\n",
-    "    return merge_tables_candidates"
+    "    return merge_tables_candidates, table_concision_list"
    ]
```

Comment:
what does table_concision_list mean? can you please help explain a bit?

Reply:
Fixed.